htmresearch/frameworks/nlp/classify_htm.py

```diff
@@ -106,7 +106,7 @@ def inferToken(self, token, reset=0, returnDetailedResults=False,
     sensor = self.sensorRegion.getSelf()
     sensor.addDataToQueue(token,
                           categoryList=[None],
-                          sequenceId=-1, reset=0)
+                          sequenceId=-1, reset=reset)
```

Comment:
Shouldn't we now remove the ["manual" reset at line 121](https://github.com/numenta/nupic.research/pull/478/files#diff-fff4bf9be700b351827d0b04ac0a0190R121)?